### PR TITLE
This is better than the old gnome term update

### PR DESCRIPTION
**Pull request by NamesCode**

### CODE REVIEW

File: Frappe/palette [hunk 1]
```diff
@@ -0,0 +1 @@
+['#51576d', '#e78284', '#a6d189', '#e5c890', '#8caaee', '#ca9ee6', '#f4b8e4', '#c6d0f5', '#737994', '#e78284', '#a6d189', '#e5c890', '#8caaee', '#ca9ee6', '#f4b8e4', '#c6d0f5']
```
Comment (bleacheda) [inline Frappe/palette:1]: according to https://github.com/catppuccin/catppuccin/blob/main/docs/style-guide.md#regular-colours, shouldn't the color scheme be this:
['#51576D', '#EA8385', '#8CAF77', '#E4C78F', '#8BA9ED', '#F3B7E3', '#7FC6BC', '#9AA3C1', '#545A6F', '#EA8385', '#AAD78C', '#E4C78F', '#8BA9ED', '#F3B7E3', '#78B8B0', '#A3ABCC']

File: Latte/palette [hunk 2]
```diff
@@ -0,0 +1 @@
+['#bcc0cc', '#d20f39', '#40a02b', '#df8e1d', '#1e66f5', '#8839ef', '#ea76cb', '#4c4f69', '#9ca0b0', '#d20f39', '#40a02b', '#df8e1d', '#1e66f5', '#8839ef', '#ea76cb', '#4c4f69']
```
Comment (bleacheda) [inline Latte/palette:1]: according to https://github.com/catppuccin/catppuccin/blob/main/docs/style-guide.md#regular-colours, shouldn't the color scheme be this:
['#5C5F77', '#D40E39', '#40A12A', '#DE8D1C', '#1D66F8', '#E673C8', '#169097', '#ABAFBD', '#6C6F86', '#D80E3A', '#3E9D29', '#DE8D1C', '#1D65F4', '#E975CA', '#169097', '#BBBFCB']

File: Macchiato/palette [hunk 3]
```diff
@@ -0,0 +1 @@
+['#494d64', '#ed8796', '#a6da95', '#eed49f', '#8aadf4', '#c6a0f6', '#f5bde6', '#cad3f5', '#6e738d', '#ed8796', '#a6da95', '#eed49f', '#8aadf4', '#c6a0f6', '#f5bde6', '#cad3f5']
```
Comment (bleacheda) [inline Macchiato/palette:1]: according to https://github.com/catppuccin/catppuccin/blob/main/docs/style-guide.md#regular-colours, shouldn't the color scheme be this:
['#474B61', '#EC8796', '#A7DC96', '#F5DAA3', '#8DB2FB', '#F8BFE9', '#8EDBD0', '#B8C0E0', '#5F647D', '#F98E9E', '#A5D994', '#C7B48A', '#8BAFF7', '#F8BFE9', '#80C4BB', '#A6AFCD']

File: Mocha/palette [hunk 4]
```diff
@@ -0,0 +1 @@
+['#45475a', '#f38ba8', '#a6e3a1', '#f9e2af', '#89b4fa', '#cba6f7', '#f5c2e7', '#cdd6f4', '#6c7086', '#f38ba8', '#a6e3a1', '#f9e2af', '#89b4fa', '#cba6f7', '#f5c2e7', '#cdd6f4']
```
Comment (bleacheda) [inline Mocha/palette:1]: according to https://github.com/catppuccin/catppuccin/blob/main/docs/style-guide.md#regular-colours, shouldn't the color scheme be this:
['#45475A', '#E0819D', '#8CBD8A', '#E5D1A3', '#7498D2', '#CDA6C5', '#88D0C5', '#BCC4E0', '#525568', '#CB7992', '#A7E6A2', '#E5D1A3', '#7498D2', '#FCC7EE', '#92E0D3', '#A7AFCA']

**Comment by nekowinston** [comment]
Properly implemented via https://github.com/catppuccin/gnome-terminal/commit/8400cbed98eeefc08da5dd1456107da0d49fc665

**Comment by NamesCode** [comment]
Thank you @bleacheda, Completely forgot about this, all credit goes to you. 👍

**Comment by nekowinston** [comment]
Well I actually rewrote my install script I wrote on September 3rd:
https://github.com/catppuccin/gnome-terminal/blob/b0e7f677a58ebfc0a0b8649d809cbed16c0f4493/install.sh#L27-L42
https://github.com/catppuccin/gnome-terminal/blob/8400cbed98eeefc08da5dd1456107da0d49fc665/install.py#L64-L81

**Pull request closed without merging**